run_tests_CLI/backend.txt

```diff
@@ -0,0 +1 @@
+numpy
```

Comment:
hey! thanks for contributing! the `run_tests_CLI` folder should not be changed at all. you should be implementing the test for the function in the corresponding file in `ivy_tests`, thanks!